podium-gateway/src/main/webapp/main.ts

```diff
@@ -19,6 +19,6 @@ if (!DEBUG_INFO_ENABLED) {
 }
 
 platformBrowserDynamic()
-    .bootstrapModule(PodiumGatewayAppModule, { preserveWhitespaces: true })
+    .bootstrapModule(PodiumGatewayAppModule, { preserveWhitespaces: false })
```

Comment:
Please revert, this breaks spacing of buttons, e.g.:
![Podium button spacing](https://user-images.githubusercontent.com/7373499/116706171-9bab0b80-a9cd-11eb-9c3f-4175e3f943e7.png)
The issue with the request type icons in the request list can be solved by removing `white-space: pre` from request-overview.scss.